lib/srv/app/aws/handler.go

```diff
@@ -0,0 +1,243 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package appaws
```

Comment:
Nit: Did you intend to name this package `appaws`? I'd consider just naming `aws` after the folder name and letting the importer use import alias if they wish to rename. I always find it confusing when a package has a different name than its directory name.